framework/include/base/Factory.h

```diff
@@ -136,7 +148,8 @@ class Factory
   template<typename T>
   void reg(const std::string & obj_name)
   {
-    /**
+
+    /*
```

Comment:
This should not be in Doxygen, this is a comment about specific code within the method.